modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -5,17 +5,19 @@
 [id="troubleshooting-disabling-autoreboot-mco_{context}"]
 = Disabling the Machine Config Operator from automatically rebooting
 
-When configuration changes are made by the Machine Config Operator (MCO), {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when a `kube-apiserver-to-kubelet-signer` certificate authority (CA) is rotated, or manual, an {op-system} node reboots automatically unless it is paused.
+When configuration changes are made by the Machine Config Operator (MCO), {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic or manual, an {op-system} node reboots automatically unless it is paused.
 
 [NOTE]
 ====
 The following modifications do not trigger a node reboot:
 
-* changes to the SSH key in the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter of a machine config
-* changes to the global pull secret or pull secret in the `openshift-config` namespace
-* changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object
+* When the MCO detects any of the following changes, it applies the changes and uncordons the node: 
```

Comment:
This is correct, but I don't think we necessarily need to mention the uncordon part. For the following no-drain no-reboot updates, the uncordon doesn't do anything (a no-op). If you cordoned the node manually as a user, the node would simply not undergo the update.

Maybe we should clarify with: "When the MCO detects any of the following changes, it applies the update without draining or rebooting the node."

Reply:
@yuqi-zhang That line ^^ 14, applies to the `registry.conf` changes. I reorganized the material a bit in one topic, but not the other (for some reason, maybe I didn't save). For `registries.conf`, I have:

 _When the MCO detects changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object, it drains the corresponding nodes, applies the changes, and uncordons the nodes._

Reply:
That sounds good for the registries.conf changes. Minor point either way for non-registries.conf changes, since the uncordon technically does happen